### PR TITLE
CI starter: add dockerfile for submit to g registry by gh action

### PR DESCRIPTION
**Pull request by Douglasacost**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+COPY ./ /app
+
+RUN apk add --no-cache python3 py3-pip make g++\
+    && rm -rf /var/cache/apk/*
+RUN cd app\
+    && yarn install\
+    && yarn codegen && yarn build
+
+HEALTHCHECK --interval=3s --timeout=5s --retries=10 \
+    CMD curl -f http://subquery-node:3000/ready || exit 1
```
Comment (secretfader) [inline Dockerfile]:
```suggestion
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+COPY ./ /app
+
+RUN apk add --no-cache python3 py3-pip make g++\
+    && rm -rf /var/cache/apk/*
+RUN cd app\
+    && yarn install\
+    && yarn codegen && yarn build
+
+HEALTHCHECK --interval=3s --timeout=5s --retries=10 \
```
Comment (secretfader) [inline Dockerfile]:
```suggestion
```

**Review by secretfader — CHANGES_REQUESTED** [review]
You can remove the `HEALTHCHECK` statement in `Dockerfile`, but that's all. Everything else looks great.

The reason we don't need that is any orchestration system (Docker Compose, Kubernetes, etc.) would probably override the healthcheck externally. I believe we'd have more occasions where the static DNS host path set in the image (`subquery-node`) wouldn't resolve, than where it worked correctly.

**Comment by Douglasacost** [comment]
> You can remove the `HEALTHCHECK` statement in `Dockerfile`, but that's all. Everything else looks great.
> 
> The reason we don't need that is any orchestration system (Docker Compose, Kubernetes, etc.) would probably override the healthcheck externally. I believe we'd have more occasions where the static DNS host path set in the image (`subquery-node`) wouldn't resolve, than where it worked correctly.

It's done. Also removed and "bad" relation that can become to have an blocker issue

**Review by secretfader — APPROVED** [review]

**Pull request merged**